templates/zerver/landing_nav.html

```diff
@@ -27,7 +27,7 @@
                 <a href="/integrations/">Integrations</a>
             </li>
             <li on-page="help">
```

Comment:
Should this `on-page` thing change?

Reply:
It should actually be removed. The change/removal is inert. This is fixed.